PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -620,7 +620,8 @@ void BuildCommandBuffers()
             // Post-transparency stack
             int tempRt = -1;
             bool forceNanKillPass = (!m_NaNKilled && stopNaNPropagation && RuntimeUtilities.isFloatingPointFormat(sourceFormat));
-            if (RequiresInitialBlit(m_Camera, context) || forceNanKillPass)
+            bool vrSinglePassInstancingEnabled = context.stereoActive && context.numberOfEyes > 1 && context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassInstanced;
+            if (!vrSinglePassInstancingEnabled && (RequiresInitialBlit(m_Camera, context) || forceNanKillPass))
             {
```

Comment:
Have you tested this in LWRP (19.2) or URP with SPI? Otherwise, LGTM

Reply:
Good call Mike. 

I've just checked URP with PPv2 to make sure it works and Mock HMD SPI keeps working with these changes and the UNITY_PS4 added define makes PSVR URP with PPv2 work as well when it appears it didn't before so that is extra good. 

Debugging it and BuildCommandBuffers where these changes are doesn't get called when using URP (and probably LWRP) but those call into the Render() method directly from the URP code itself.

Reply:
Ok, thanks, Robin for trying out in URP.